applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -167,3 +174,18 @@ def Initialize(self):
 
     def AdvanceInTime(self, current_time):
         raise Exception("AdvanceInTime is not implemented. Potential Flow simulations are steady state.")
+
+    def Predict(self):
+        self.solver.Predict()
+
+    def FinalizeSolutionStep(self):
+        (self.solver).FinalizeSolutionStep()
+
+    def SolveSolutionStep(self):
+        is_converged = self.solver.SolveSolutionStep()
+        if not is_converged:
+            msg  = "Fluid solver did not converge for step " + str(self.main_model_part.ProcessInfo[KratosMultiphysics.STEP]) + "\n"
+            msg += "corresponding to time " + str(self.main_model_part.ProcessInfo[KratosMultiphysics.TIME]) + "\n"
+            KratosMultiphysics.Logger.PrintWarning("PotentialFlowSolver",msg)
+            return is_converged
```

Comment:
For these functions, I would try to use the ones from the FluidSolver as much as possible (the FluidSolver is the PotentialSolver parent). 
Which error do you get if you use the parents method (i.e not defining Predict() FinalizeSolut..() SolveSol..() here)?

Reply:
the fluid solver also check if the time buffer is initialized

Thinking abt it I think setting the `min_buffer_size` to 1 should have the same effect and should be sufficient for potential flow right @inigolcanalejo @marcnunezc ?
=> then you don't have to override those functions

Reply:
Yes, that should work.

Reply:
Actually I think that we changed this when we made the PotentialFlowApplication to depend on the FluidDynamicsApplication.